### DISCUSSION TITLE
隧道持续启动失败, 已关闭该隧道

### DISCUSSION THREAD

**Discussion by cxa777** [natfrp/wiki#417] [使用问题]
2024/01/22 18:56:51 W Tunnel/cxa 无法启动 frpc: fork/exec C:\Program Files\SakuraFrpLauncher\frpc.exe: Access is denied.

**Comment by cxa777** [comment]
帮帮忙吧兄弟们

**Reply by IAmTPC** [reply]
是不是你的ip被禁了

**Comment by InstantNoodles** [comment]
https://doc.natfrp.com/launcher/antivirus.html 试试文档里的方法呢
